### ISSUE TITLE
New trainers can't use goals

### ISSUE THREAD

**Issue by JayTurnr** [PokemonGoEastKent/red-trainerdex#5]
If the trainer's oldest update is fewer days away from today than the second argument on getDiff(), you will get IndexError: list index out of range.
```
Traceback (most recent call last):
  File "lib/discord/ext/commands/core.py", line 50, in wrapped
    ret = yield from coro(*args, **kwargs)
  File "/opt/red/cogs/trainer.py", line 196, in xp
    embed = await self.updateCard(trainer)
  File "/opt/red/cogs/trainer.py", line 97, in updateCard
    totalDiff = await self.getDiff(trainer, 7)
  File "/opt/red/cogs/trainer.py", line 65, in getDiff
    reference = reference[0]
IndexError: list index out of range
```

**Comment by JayTurnr** [comment]
Solution will be something along the lines of:
```
try:
    #existing code
except IndexError:
    #check if any updates exist
    if oldest update < 6 days 21 hours ago:
        reference = oldest update
    #if no updates exist, return None or an effective none which won't give more errors, log error.
```

**Comment by JayTurnr** [comment]
Fixed with b9612fd
